DSharpPlus/Net/Abstractions/Rest/RestChannelPayloads.cs

```diff
@@ -297,4 +298,14 @@ internal sealed class RestModifyStageInstancePayload
         [JsonProperty("privacy_level")]
         public Optional<PrivacyLevel> PrivacyLevel { get; set; }
     }
+
+    internal sealed class RestBecomeStageSpeakerInstancePayload
+    {
+        [JsonProperty("channel_id")]
+        public ulong ChannelId { get; set; }
+        [JsonProperty("request_to_speak_timestamp")]
+        public DateTime? RequestToSpeakTimestamp { get; set; }
```

Comment:
Considering this is nullable, `NullValueHandling = NullValueHandling.Ignore` should be applied for consistency.